rpc/send_test.go

```diff
@@ -156,6 +156,31 @@ func TestUnretryableError(t *testing.T) {
 	}
 }
 
+// TestClientNotReady verifies that Send gets an RPC error when a client
+// does not become ready.
+func TestClientNotReady(t *testing.T) {
+	rpcContext := createNewTestRPCContext(t)
+
+	opts := Options{
+		N:               1,
+		Ordering:        OrderStable,
+		SendNextTimeout: 1 * time.Second,
+		Timeout:         1 * time.Second,
```

Comment:
Changed to `Nanosecond`. 

Agreed. Will keep that in mind. Maybe will build a something like a fake client/server that does directly method call and removes unnecessary startup if tests are getting too slow.